app/controllers/api/v1/ratings_controller.rb

```diff
@@ -0,0 +1,49 @@
+module Api
+  module V1
+    class RatingsController < ApplicationController
+      before_action :set_rating, only: %i[show update destroy]
+
+      def index
```

Comment:
does it make sense to display all ratings?

Reply:
total rating of movie, I think I'll put it in a separate service